src/source/examples/get-feed-import-id-detail-v2-404.rst

```diff
@@ -1,6 +1,6 @@
 .. code-block:: javascript
 
-    GET /api/sellside/feed/import/123/detail
+    'GET /api/sellside/feed/import/123/detail'
```

Comment:
these parts should not have quotes, since they are raw headers

Reply:
That is somehow unrelated to the main topic. I wanted to fix the error message on this file. "import" in the path seems to be a keyword of this sphinx/javascript lever. Quoting was the quickest way to get rid of the error.

Reply:
aaah that's why these errors exist! that's a nice find :D 
Still, this means the example shows with quotes in the docs, no? In that case I'd rather have the warning...